### PR TITLE
migrate kb articles to cluster deployment and production readiness

### PR DESCRIPTION
**Pull request by axfelix**
3 more KB migrations:

https://temporalio.atlassian.net/browse/EDU-1275

https://temporalio.atlassian.net/browse/EDU-1283

https://temporalio.atlassian.net/browse/EDU-1280

### CODE REVIEW

File: docs-src/prod-readiness-context/platform-defaults.md [hunk 1]
```diff
@@ -1,41 +1,35 @@
 ---
-slug: temporal-platform-limits-sheet
-title: Temporal Platform limits sheet
+id: platform-defaults
+title: Temporal Platform Defaults
```
Comment (djmagee) [inline docs-src/prod-readiness-context/platform-defaults.md:3]:
```suggestion
title: Temporal Platform defaults
```

File: docs-src/prod-readiness-context/platform-defaults.md [hunk 2]
```diff
@@ -1,41 +1,35 @@
...
   - limits
-date: 2023-09-11T00:00:00Z
 ---
 
 :::caution
 
-This page will be removed in the near future.
-
-For the latest information on the Temporal Cloud limits, see [Temporal Clouds limits sheet](/cloud/operating-envelope#temporal-platform-limits-sheet).
+For the latest information on the Temporal Cloud limits, see [Temporal Cloud limits sheet](/cloud/operating-envelope#default-limits).
```
Comment (djmagee) [inline docs-src/prod-readiness-context/platform-defaults.md:14]:
```suggestion
For the latest information on the Temporal Cloud default limits, see [What are the Temporal Cloud default limits?](/cloud/operating-envelope#default-limits)
```

File: docs-src/concepts/what-is-an-update.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ An Update has four phases.
 
 1. **Admission.** The Temporal Cluster first validates Update submissions against the configured resource usage limits.
    For example, limits apply to concurrent requests and requests per second.
-   See the [Temporal Platform limits sheet](/kb/temporal-platform-limits-sheet) for more details.
+   See the [Temporal Platform limits](/production-readiness/tuning#platform-defaults) for more details.
```
Comment (djmagee) [inline docs-src/concepts/what-is-an-update.md:37]:
```suggestion
   For more details, see the [Temporal Platform defaults](/production-readiness/tuning#platform-defaults).
```

File: docs-src/cloud/prometheus-grafana-setup-cloud.md [hunk 4]
```diff
@@ -252,7 +252,7 @@ If you see metrics on the scrape endpoints, but Prometheus shows your targets ar
 Verify your Prometheus configuration and restart Prometheus.
 
 If you’re running Grafana as a container, you can set your SDK metrics Prometheus data source in your Grafana configuration.
-See the example Grafana configuration described in the [Prometheus and Grafana setup for open-source Temporal Cluster](/kb/prometheus-grafana-setup) KB article.
+See the example Grafana configuration described in the [Prometheus and Grafana setup for open-source Temporal Cluster](/production-readiness/monitoring#prometheus-grafana-setup) article.
```
[the remark this answers is not in the document]
Comment (axfelix) [reply docs-src/cloud/prometheus-grafana-setup-cloud.md:255]: Yeah, good call! I like the more specificity!

File: docs-src/cluster-ops-context/intro.md [hunk 5]
```diff
@@ -1,18 +1,191 @@
...
+The Temporal Server should be available on `localhost:7233` and the Temporal Web UI should be accessible at [`http://localhost:8233`](http://localhost:8233/).
+
+The server's startup configuration can be customized using command line options.
+For a full list of options, run:
+
+```bash
+temporal server start-dev --help
+```
+
+### Docker & Docker Compose
```
[the remark this answers is not in the document]
Comment (axfelix) [reply docs-src/cluster-ops-context/intro.md:104]: I'll have to update the anchor in several places if we make this change... not sure if our style guide is anti-ampersand but putting a pin in it for now!

**Review by rachfop — APPROVED** [review]

**Review by flossypurse — CHANGES_REQUESTED** [review]
blocking to prevent accidental merge.

**Comment by axfelix** [comment]
Now also handles https://temporalio.atlassian.net/browse/EDU-1278 and https://temporalio.atlassian.net/browse/EDU-1279

**Comment by axfelix** [comment]
ES6 migration page is now at https://community.temporal.io/t/migrating-visibility-data-from-elasticsearch-6/9970, as discussed, which addresses https://temporalio.atlassian.net/browse/EDU-1336.

**Comment by flossypurse** [comment]
Moved off of a forked branch onto this: https://github.com/temporalio/documentation/pull/2463

**Comment by flossypurse** [comment]
commits merged in this PR: https://github.com/temporalio/documentation/pull/2463

**Pull request closed without merging**